tests/unit/remoteworker/test_remote_worker.py

```diff
@@ -212,3 +213,98 @@ def test_download_and_extract_file_when_download_fails(self, mock_logger):
 
         mock_logger.assert_called_with(expected)
         self.assertFalse(os.path.exists(self.download_location))
+
+
+class ProcessSubmissionMessageTest(BaseTestClass):
+    def setUp(self):
+        super(ProcessSubmissionMessageTest, self).setUp()
+        self.queue_name = "evalai_submission_queue"
+        self.submission_data_dir = "tmp/test-dir/submission_{}".format(self.submission_pk)
+        self.input_file = "user_annotation_file.txt"
+        self.remote_evaluation = "Test Remote Evaluation"
+
+        # Creating sample outputs with minimal data for mocked functions
+        self.message = {
+            "challenge_pk": self.challenge_pk,
+            "phase_pk": self.challenge_phase_pk,
+            "submission_pk": self.submission_pk,
+        }
+        self.submission = {"id": self.submission_pk, "input_file": self.input_file}
```

Comment:
@nikochiko can you add every new key value pair on a newline. Just to make the code clean.

Reply:
Sure! Will do it right away!